types/block_test.go

```diff
@@ -130,26 +117,26 @@ func cidFromString(input string) (cid.Cid, error) {
 func TestDecodeBlock(t *testing.T) {
 	tf.UnitTest(t)
 
-	newSignedMessage := NewSignedMessageForTestGetter(mockSigner)
 	t.Run("successfully decodes raw bytes to a Filecoin block", func(t *testing.T) {
 		addrGetter := address.NewForTestGetter()
 
 		c1, err := cidFromString("a")
 		assert.NoError(t, err)
 		c2, err := cidFromString("b")
 		assert.NoError(t, err)
+		cM, err := cidFromString("messages")
+		assert.NoError(t, err)
```

Comment:
Maybe it would just be better to force the caller to always provide a string to cbor encode, make `cidFromString` an export/test helper and get rid of `SomeCid` entirely.

Reply:
#3193